superset-frontend/packages/superset-ui-core/src/query/types/QueryResponse.ts

```diff
@@ -76,6 +76,8 @@ export interface ChartDataResponseResult {
     | 'scheduled'
     | 'success'
     | 'timed_out';
+  from_dttm: number | null;
+  to_dttm: number | null;
```

Comment:
It appears the `ChartDataResponseResult` schema is pretty badly out of sync with this. For example, the following appear to be missing or have an incorrect name:
- here we have `cache_dttm`, on `ChartDataResponseResult` we have `cached_dttm`
- `colnames` and `coltypes` are missing on `ChartDataResponseResult`.
- `from_dttm` and `to_dttm` should also be added to `ChartDataResponseResult`.
Let's try to make sure these are in sync.